newsroom/design.py

```diff
@@ -39,11 +50,14 @@ async def detail():
     return await render_template("wire_item.html", item=item, previous_versions=previous_versions)
 
 
-@blueprint.route("/design/")
-async def index():
+@blueprint.endpoint("/design/", auth=False)
+async def index(args: None, params: None, req: Request):
```

Comment:
nit: you probably don't need all these arguments
```suggestion
async def index():
```